veldymanov/tile-shop-be
# Task 7

Task 7.1 - DONE

Task 7.2 - DONE

Task 7.3 - DONE
- FE PR: https://github.com/veldymanov/tile-shop-fe/pull/8
- WebSite: https://d2lvjuwl8mt9u1.cloudfront.net/

Additional Tasks:
- 401/403 alerts added
- Cognito authorization is implemented

Testing:
1. Go to https://d2lvjuwl8mt9u1.cloudfront.net/admin/products
2. Try to upload one of the testing files [products.zip](https://github.com/veldymanov/tile-shop-be/files/6657024/products.zip). You should get 'Unathorized' alert.
3. Then click on 'User Profile' icon, choose 'Login' and then sign up. 
4. You will get Verification Code to you email.
5. Try p.1, p.2 again. Wait for a while and reload page.
